code/game/machinery/traitorous_cloner.dm

```diff
@@ -0,0 +1,360 @@
+//Experimental cloner; clones a body regardless of the owner's status, letting a ghost control it instead
+
+/obj/machinery/clonepod/traitorous
+	name = "experimental cloning pod traitorous"
```

Comment:
This is a really weird name to have

Reply:
The original bounty creator is on creating names, so this is pointless